.sync.yml

```diff
@@ -2,4 +2,6 @@
 appveyor.yml:
   delete: true
 .github/workflows/ci.yml:
+  beaker_hosts: 'host1:shard.ma;host2:slave,router.a'
   beaker_facter: 'mongodb_repo_version:MongoDB:4.4,5.0,6.0,7.0'
+  
```

Comment:
before we merge this, please fix the broken newline.

Reply:
I'm going to fix it in #743 and will close this PR without merge when I'm done with all the gha changes.